### DISCUSSION TITLE
Buyers Currency Payment Info or Payment Exchange Rate

### DISCUSSION THREAD

**Discussion by milonic** [etsy/open-api#1163] [Ideas]
Hi Guys,

Just had an email response from Etsy Support, regarding a question I had about obtaining the amount a buyer has paid in their currency as opposed to the amount received in the shops currency. 

Basically, I'd like to know how much a buyer in a different country has paid for items using the API. The response I had from Etsy Support is as follows:

_At this time, we do not share out the exact amount the buyer paid in their currency through our API nor through the shop manager/Etsy UI._ 

However, this is not true. If I visit Shop Manager, I can print an invoice and get the exact information I need. Unfortunately, this is in PDF format but I'd like this to be available from the API.

Can this feature be added to the API?

I'd assume the end point: getShopPaymentByReceiptId would provide this as it does state the buyers currency but all values are in the shops currency.

The email also suggested I use a third party system to obtain an exchange rate. But simply will not work as I need to report back to the buyer with order details so that they can download digital products from our own website. And we do this because Etsy’s system does not always send out order emails to buyers. Nor can buyers download digital products using the app. 

All we need is exchange rate adding to the getShopPaymentByReceiptId, getPayments and getShopReceipt end points.

Can this be added to the “To Do” list?

**Comment by Gareth064** [comment]
Hey @milonic did you respond to the email from Etsy with everything you just told us in this discussion? Etsy to work from this repo, so unfortunately asking for it to be added to a To Do list will do nothing here. You should respond to the email asking Etsy to look into implementing it. Explaining your use case as you have above.

**Reply by milonic** [reply]
Yes Gareth, I did.

Toward the bottom of their email it suggested I post something on Github but it might have been standard Etsy spiel.

Anyways, I thought it would do no harm and I might get support from others and actually get something done about it.

Informing a buyer: "Here are your order details, but we can't itemize things because we've no idea how much you paid" isn't very good really is it.

We can live in hope!

**Reply by Gareth064** [reply]
Your the second person that has had Etsy tell them to post something in here... but from what I can tell (I subscribe to every action that happens here) Etsy Developers never posts or answer anything in here anymore. They have went complete silent, except for when @nateswart does a post to say if there are any breaking changes inbound.
